pypeit/edgetrace.py

```diff
@@ -1207,6 +1207,9 @@ def save(self, outfile=None, overwrite=True, checksum=True, float_dtype='float32
             _outfile = _outfile[:_outfile.rfind('.')] 
             compress = True
 
+        # First check if a trace is available
+        if self.spat_cen is None:
+            msgs.error("No trace information available")
```

Comment:
There's an `@property` method called `is_empty`.  Does it work to replace `if self.spat_cen is None` with `if self.is_empty`?  If not, we should fix `is_empty`.